release-notes/swan-lake-beta1-release-note.md

```diff
@@ -124,6 +149,14 @@ location = "UK"
 username = "Jane"
 location = "US"
 ```
+#### Other Changes
+
+- Taint analyzer is disabled and `--taint-check` compiler flag is removed. The annotations `@tainted` and `@untainted` were left intact as place holders for backword compatibility.
```

Comment:
Can't we include this also under breaking changes?

```suggestion
- Taint analyzer is disabled and the `--taint-check` option is removed. The annotations `@tainted` and `@untainted` were left intact as place holders for backward compatibility.
```

Reply:
Fixed https://github.com/ballerina-platform/ballerina-release/pull/416/commits/4699828ad153bcf58a0e588b0524654fbe0a5db4